workflows/ZWEWRF06.properties

```diff
@@ -0,0 +1,59 @@
+# zowe_install_path
+# Label: Install Path
+# Abstract: Path for the Zowe runtime
+# Category: Installation
+# Description:
+#  Specify the path that was used for the ZOWE runtime installation
+zowe_install_path=
+
+# zowe_dsn_prefix
+# Label: Zowe Data Set Prefix
+# Abstract: Data Set HLQ for the Zowe runtime installation
+# Category: Installation
+# Description:
+#  Specify the data set high level qualifier that was used for the Zowe runtime installation
+zowe_dsn_prefix=
+
+# xmem_prefix
+# Label: Cross Memory Server DSN Prefix
+# Abstract: Please specify the HLQ that will be used for the XMEM
+# Category: Installation
+# Description:
+#  High Level Qualifier that will be used to install and configure the Cross Memory Server
+xmem_prefix=
+
+# xmem_dasd
+# Label: Cross Memory Server DASD
+# Abstract: Please specify the DASD that will be used for the XMEM data sets.
+# Category: Installation
+# Description:
+#  Please specify the DASD that will be used for the XMEM data sets.
+#  Specify the DASD in either of the following formats:
+#  VOL=SER=<i>volume</i> for non-SMS managed data sets
+#  STORCLAS=<i>storage_class</i> for SMS managed data sets
+# Choices: VOL=SER=??????,STORCLAS=????????
+xmem_dasd=
+
+# proclib
+# Label: Procedure Library
+# Abstract: Please specify the procedure library for the started task
+# Category: Installation
+# Description:
+#  Specify the PROCLIB data set name that will be used to store the XMEM STC
+proclib=
+
+# parmlib
+# Label: Parameter Library
+# Abstract: Please specify the active PARMLIB
+# Category: Parmlib
+# Description:
+#  Please specify the active system PARMLIB that will be used to specify the execution of the STC.
```

Comment:
Would we want to mention that this is the PARMLIB with the SCHEDxx member used by the system? The current description is not very clear and this PARMLIB can be confused for the PARMLIB used by the cross-memory server itself.

So maybe something like that?
```
Please specify the PARMLIB of your active SCHEDxx member.
```